modules/distributor/distributor.go

```diff
@@ -87,6 +87,11 @@ var (
 		Name:      "distributor_bytes_received_total",
 		Help:      "The total number of proto bytes received per tenant",
 	}, []string{"tenant"})
+	metricIngressBytes = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Being a bit picky, it seems we have a convention where the unit of measurement goes first:

`metricBytesIngested`

Although this seems incorrect, following Prometheus naming conventions. https://prometheus.io/docs/practices/naming/#metric-names


Instead of Ingress and received which may be confusing maybe we could use something like:


`metricBytesRecievedRaw`
`distributor_bytes_received_raw_total`

Reply:
along with `distributor_bytes_received_raw_total` I want to throw in two more names into the mix:
- `tempo_distributor_bytes_attempted_total` since these are the bytes that are attempted (might be ingested or not)
- `distributor_bytes_received_unfiltered_total` since we are capturing the bytes before ingestion limits are applied (i.e. filtering the data)

Reply:
Being even _nitpickier_, `ingress bytes` can be considered the measurement unit in this metric. If we wanted a verb after the unit like in `distributor_bytes_received_total` it could be `distributor_ingress_bytes_received_total` or `distributor_bytes_ingressed_total`.

Ideally, we would rename `distributor_bytes_received_total` to `ingested` and use `distributor_bytes_received_total` for this new metric.

I don't have a strong opinion on this, I'm ok to go with whichever name sounds more clear for the team.

Reply:
I don't want to bikeshade on the name and okay with `tempo_distributor_ingress_bytes_total` as well.